gslib/commands/mb.py

```diff
@@ -158,12 +157,6 @@
                          "enforced", objects in this bucket cannot be made
                          publicly accessible. Default is "inherited".
 
```

Comment:
@rebecca-pete Can you please take a look at the help text for -l command abobe and let me know if we need to specify anything about dual regions there. My thought was that we probably don't as the link in the help text https://cloud.google.com/storage/docs/locations  would already cover that information.